spec/webmock/alert_spec.rb

```diff
@@ -42,5 +42,30 @@
         expect(WebMock).to have_requested(:put, "#{host}/api/alert")
       end
     end
+
+    it 'Fetches Alert' do
```

Comment:
this test fails without the code from https://github.com/mrgutie8/metabase-ruby-rest-api/pull/56.  They should be in one PR so the test pass and the lib has a test with it